sdk/core/ci.yml

```diff
@@ -46,6 +46,7 @@ extends:
       safeName: AzureCore
       triggeringPaths:
       - /sdk/resourcemanager/
+      - /eng/
```

Comment:
Just curious - what is the difference between the trigger path here and the other ones above for `eng/`?